### ISSUE TITLE
Make disputed order notice copy clear, readable, and actionable

### ISSUE THREAD

**Issue by haszari** [Automattic/woocommerce-payments#7568]
### Describe the bug

Follow up to #6354 (and some other more recent changes? The screenshots in that issue don't match current code)

The current iteration of the disputed order notice is hard to read, and is rendered all in bold. The goal of this issue is to tidy it up so it drives merchants to follow up disputes.

- Only use bold for the first sentence, clearly stating the dispute amount and reason.
- Switch from lengthy, redundant ~~`The customer provided the following reason`~~ wording to `This order has a payment dispute for {AMOUNT} for the reason '{reason}'`.

More info from @nikkivias :

On another note— I am noticing that the text on the screenshots above is quite lengthy?

> The customer provided the following reason: [ the reason ] 

I would recommend to implement the below text and styling where the intro sentence is shorter and the only one in bold. The rest will be in normal weight. This would make the content of the notice appear less heavy and potentially at just 2 lines.

![Screenshot 2023-10-27 at 12 57 24 AM](https://github.com/Automattic/woocommerce-payments/assets/99638528/9b67fb89-66d9-4c56-a910-8fda166da0b5)

[(See the mockup on Figma)](RiEQmaKRI7u9USAI3lyZz0VX-fi-7596%3A86883)

### To Reproduce

1. Go to '…'
2. Click on '…'
3. Scroll down to …'

### Desktop (please complete the following information):

* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

### Smartphone (please complete the following information):

* Device: [e.g. iPhone6]
* OS: [e.g. iOS8.1]
* Browser [e.g. stock browser, safari]
* Version [e.g. 22]

**Comment by souravdebnath1986** [comment]
Changed priority to `high` as this could impact our dispute response rates. 

IMO there are 4 pieces of information in the order note - 
1. dispute status i.e. does it need response or is it in progress or is a final state reached
2. dispute amount 
3. dispute reason
4. dispute response date 
Combining more than one piece into a sentence is going to be hard to parse for the merchant. 

`How may we` make order note clearer so that the merchant can see all pieces of info well.... 

Example: 

> This order has a payment dispute that `Needs Response`
Amount: $120.00 USD | Reason: Purchase Unauthorised | Respond by: May 23, 2023 

Styling and wording of the status, amount, reason and response date should be consistent across dispute list and detail (including evidence submission form) views. 
cc: @rogermattic

**Comment by haszari** [comment]
To clarify @souravdebnath1986 @rogermattic  – we already have design & wording for this, it was picked up by @nikkivias when we did our UX review and this was logged as a result. 

I've included the new wording/formatting in the description, let me know if it's not clear. The wording in the description should work well for all `reason`s. Proposed wording:

> **This order has a payment dispute of {AMOUNT} for the reason '{REASON}'.** Please respond before {DEADLINE}. 

Happy to hear your input @rogermattic – but we are not waiting on design and are happy to implement & ship this based on the description.
